tls/tls_srv.c

```diff
@@ -325,7 +345,7 @@ ttls_parse_alpn_ext(TlsCtx *tls, const unsigned char *buf, size_t len)
 	 */
 
```

Comment:
I think `ttls_parse_supported_point_formats()` needs to be updated too. We dereference `buf` there as `buf[0]` without checking available length first. Also, there is a similar `buf[0]` access in `ssl_parse_supported_point_formats_ext()`.